### DISCUSSION TITLE
Why is  'fgwped' available as an interation variable?

### DISCUSSION THREAD

**Discussion by jonmaddock** [ukaea/PROCESS#2888] [Ideas]
In GitLab by @mkovari on May 31, 2023, 11:37

I am not sure why we would want to allow `fgwped` to be an interation variable.  
`fgwped = fraction of Greenwald density to set as pedestal-top density (iteration variable = 145).`  

The variables `fgwped_out` and `fgwsep_out` are also odd.  (The comment below is not mine.)

physics.f90
```
        ! This code is ODD! Don't change it! No explanation why fgwped and fgwsep
        ! must be assigned to their exisiting values!
        fgwped_out = neped/dlimit(7)
        fgwsep_out = nesep/dlimit(7)
        if(fgwped >= 0d0) fgwped = neped/dlimit(7)
        if(fgwsep >= 0d0) fgwsep = nesep/dlimit(7)


```

Any comments @jmorris-uk @ajpearcey @stuartmuldrew ?

**Comment by jmorris-uk** [comment]
I believe that they were included to allow for the I-mode runs where the pedestal density had to be lowered while the temperature remained high at the pedestal.

I don't think the option is really needed anymore and if it is needed, then probably a better implementation is required.

**Comment by mkovari** [comment]
As long as a sensible upper bound is applied this doesn't seem like a crime.  Since the volume-average density is usually an iteration variable and the Greenwald limit is applied as an upper bound, it makes sense to allow the pedestal density to vary to fit in with everything else.
